pod/perlop.pod

```diff
@@ -673,10 +673,6 @@ scalar expression that yields a string class name:
     if( $obj isa "Different::Class" ) { ... }
     if( $obj isa $name_of_class ) { ... }
 
-This is an experimental feature and is available from Perl 5.31.6 when enabled
-by C<use feature 'isa'>. It emits a warning in the C<experimental::isa>
-category.
```

Comment:
done

Reply:
Feature-gated functions, such as [`say`](https://perldoc.perl.org/functions/say) and [`fc`](https://perldoc.perl.org/functions/fc), refer to `use v5.xx;` rather than feature bundles, so I think the wording here should be consistent with that.

Reply:
Ahyes. Further wording fixes to be similar to the `say` docs.